### PR TITLE
color-code 0.8.7

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
   ==> /opt/homebrew/opt/qt@5/bin/qmake
  Usage: /opt/homebrew/opt/qt@5/bin/qmake [mode] [options] [files]
  
  QMake has two modes, one mode for generating project files based on
  some heuristics, and the other for generating makefiles. Normally you
  shouldn't need to specify a mode, as makefile generation is the default
  mode for qmake, but you may use this to test qmake on an existing project
```

**Comment by chenrui333** [comment]
relates to:
- https://bugreports.qt.io/browse/QTBUG-117225
- https://github.com/Homebrew/homebrew-core/issues/144051

**Comment by p-linnane** [comment]
Apparently the recommended build system now is `cmake`, so we can try that.

**Comment by chenrui333** [comment]
> Apparently the recommended build system now is `cmake`, so we can try that.

yeah, I was thinking of it, but there is app part :)

File: Formula/c/color-code.rb [hunk 1]
```diff
@@ -25,24 +24,21 @@ class ColorCode < Formula
...
-
-    if OS.mac?
-      prefix.install "ColorCode.app"
-      bin.write_exec_script "#{prefix}/ColorCode.app/Contents/MacOS/colorcode"
-    else
-      bin.install "colorcode"
-    end
+    system "cmake", "-S", "src", "-B", "build_cmake", *std_cmake_args
+    system "cmake", "--build", "build_cmake"
+    system "cmake", "--install", "build_cmake"
```
Comment (chenrui333) [inline Formula/c/color-code.rb:35]: do we still need this step?
```suggestion
```

Comment (p-linnane) [reply Formula/c/color-code.rb]: Good point. Forgot to remove that when debugging as we no longer need it.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**